src/AvatarList/index.js

```diff
@@ -1,11 +1,16 @@
 import React, { Component } from 'react'
-import { View, StyleSheet, Platform } from 'react-native'
+import { View, StyleSheet, Platform, ActivityIndicator } from 'react-native'
 import ImageScrollViewWeb from './ImageScrollView.web.js'
 import ImageScrollViewMobile from './ImageScrollView.js'
 import ImageScrollViewPWA from './ImageScrollView.pwa.js'
 import EmptyState from '../Shared/EmptyState'
+import PropTypes from 'prop-types'
 
 class AvatarList extends Component {
+  static contextTypes = {
+    getFlags: PropTypes.func,
+  }
```

Comment:
I'd prefer not to introduce a [legacy API](https://reactjs.org/docs/legacy-context.html) into the repo. I think it's also a bit weird for this to have to rely on context from the installing module.

Can we instead pass in the feature flag object from `runner` into the [`ObjectClass`](https://github.com/AdaloHQ/runner/blob/main/src/components/ObjectRenderer.js#L188) here and the [`ComponentClass`](https://github.com/AdaloHQ/runner/blob/main/src/components/LibraryComponent.js#L913) here and then access them like we would any other prop?